core/bin/external_node/src/config/mod.rs

```diff
@@ -363,6 +367,13 @@ impl OptionalENConfig {
         10_000
     }
 
+    const fn default_main_node_rate_limit_rps() -> NonZeroUsize {
+        match NonZeroUsize::new(100) {
+            Some(value) => value,
+            None => unreachable!(),
+        }
```

Comment:
Bikeshedding: `NonZeroUsize::new(100).unwrap()` seems to be just enough and would make you think less about the construction. Subjective, of course.